jax/experimental/jax2tf/examples/saved_model_lib.py

```diff
@@ -116,7 +116,7 @@ def convert_and_save_model(
     params)
   tf_graph = tf.function(lambda inputs: tf_fn(param_vars, inputs),
                          autograph=False,
-                         jit_compile=compile_model)
+                         experimental_compile=compile_model)
```

Comment:
We normally use tf_nightly for testing this, jax2tf in principle depends on relatively new features of TF. It may work with the 2.4.1 release, but I suspect that in some corners cases is going to fail.

Reply:
I believe it is called `experimental_compile` from TF 2.4 onwards ([see first comment here](https://www.tensorflow.org/xla/tutorials/jit_compile)). I just installed `tf-nightly` (`TF2.6.0-dev20210429`) and there the parameter is also called `experimental_compile`. Or perhaps I am misunderstanding something?

Reply:
It used to be called experimental_compile but in November it was declared non-experimental, and was renamed to jit_compile. See [here](https://github.com/tensorflow/tensorflow/commit/9df9d06e27882504b75e16e4c444ee22405340b3#diff-2b1b69303b927a484e02c7fad9fc87d0d3ff0dc22ae1da0ecd0dc935d922a23c). It [seems](https://github.com/tensorflow/tensorflow/releases) that this will be released with 2.5.0. I think that experimental_compile will be kept for backwards compatibility.

Reply:
I see, that makes sense. Indeed, I just tried it and `jit_compile` also works in tf-nightly. I've now changed it back to `jit_compile`.